src/test/java/org/broadinstitute/hellbender/utils/SimpleIntervalUnitTest.java

```diff
@@ -29,10 +29,14 @@ public void testEquality(){
         final SimpleInterval i3 = new SimpleInterval("chr1", 1, 100);
         final SimpleInterval i4 = new SimpleInterval("chr1", 2, 100);
         final SimpleInterval i5 = new SimpleInterval("chr1", 1, 200);
+        final SimpleInterval i6 = new SimpleInterval("1:1-100");
```

Comment:
Add a `DataProvider` with test cases for all supported String interval formats: `contig`, `contig:start`, `contig:start+`, `contig:start-end`, with and without embedded commas in the start/stop positions. Then add a new test case (don't embed these parsing tests within `testEquality()`) that uses this `DataProvider` and checks the values of contig/start/stop for each test case.